Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.5
+
+COPY . /app
+WORKDIR /app
+RUN pip install -e .
+
+ENV XSNIPPET_API_SETTINGS=/app/xsnippet_api/default.conf
```

Comment:
Well, we can point it to some, let's say, `/etc/xsnippet-api.conf`. In that case, from API purpose of view all we need is to drop additional config into that location. What do you think?